tests/drivers/dma/loop_transfer/src/dma.c

```diff
@@ -26,8 +26,8 @@ static __aligned(16) char tx_data[64] __used
 static __aligned(16) char rx_data[TRANSFER_LOOPS][RX_BUFF_SIZE] __used
 	__attribute__((__section__(".nocache.dma")));
 #else
-/* pad to times of 8*/
-static const char tx_data[] =
+/* this src memory shall be in RAM to support usingas a DMA source pointer.*/
+static char tx_data[] =
```

Comment:
Does this need to be aligned the way `rx_data` is?

Reply:
I did not see any restiction on src address alignment for kinetis, so I did not add it. maybe it is better t add, I will track it. Thanks